com.unity.cinemachine/Editor/Editors/InputAxisControllerEditor.cs

```diff
@@ -57,28 +57,43 @@ public override void OnInspectorGUI()
                 }
                 else
                 {
+#pragma warning disable CS0219 // Variable is assigned but its value is never used
```

Comment:
There is no need for this #pragma here, because the ifs at line 84 and 89 use the value, so we won't get a warning.
Also, the pragma here with the other #ifs cause a bit of a clutter to my eyes. That's why I checked :)